repositories.bzl

```diff
@@ -84,7 +84,7 @@ IO_GRPC_GRPC_JAVA_OVERRIDE_TARGETS = {
     "io.grpc:grpc-protobuf": "@io_grpc_grpc_java//protobuf",
     "io.grpc:grpc-protobuf-lite": "@io_grpc_grpc_java//protobuf-lite",
     "io.grpc:grpc-rls": "@io_grpc_grpc_java//rls",
-    "io.grpc:grpc-services": "@io_grpc_grpc_java//services",
+    "io.grpc:grpc-services": "@io_grpc_grpc_java//services:services_maven",
     "io.grpc:grpc-stub": "@io_grpc_grpc_java//stub",
     "io.grpc:grpc-testing": "@io_grpc_grpc_java//testing",
     "io.grpc:grpc-xds": "@io_grpc_grpc_java//xds",
```

Comment:
Should this also be `xds:xds_maven` ?

Reply:
Yes, it should, although since `xds:xds` does exist it wouldn't commonly cause trouble. I'll change that tomorrow.

Reply:
hello @ejona86 , follow up warning will be fixed in v1.49.0 ?

```
DEBUG: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/java_grpc_library.bzl:76:14: in srcs attribute of @io_grpc_grpc_java//services:_health_java_grpc: Proto source with label @io_grpc_grpc_proto//:health_proto should be in same package as consuming rule
WARNING: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/services/BUILD.bazel:165:18: in _java_grpc_library rule @io_grpc_grpc_java//services:_reflection_java_grpc: target '@io_grpc_grpc_java//services:_reflection_java_grpc' depends on deprecated target '@io_grpc_grpc_proto//:reflection_proto_deprecated': Use :reflection_proto instead
WARNING: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/services/BUILD.bazel:165:18: in _java_grpc_library rule @io_grpc_grpc_java//services:_reflection_java_grpc: target '@io_grpc_grpc_java//services:_reflection_java_grpc' depends on deprecated target '@io_grpc_grpc_proto//:reflection_java_proto_deprecated': Use :reflection_java_proto instead
DEBUG: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/java_grpc_library.bzl:76:14: in srcs attribute of @io_grpc_grpc_java//services:_reflection_java_grpc: Proto source with label @io_grpc_grpc_proto//:reflection_proto_deprecated should be in same package as consuming rule
WARNING: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/services/BUILD.bazel:105:13: in java_library rule @io_grpc_grpc_java//services:reflection: target '@io_grpc_grpc_java//services:reflection' depends on deprecated target '@io_grpc_grpc_proto//:reflection_java_proto_deprecated': Use :reflection_java_proto instead
DEBUG: /private/var/tmp/_bazel_shaoxixu/241205fdee0dfc3430ee79ef9f505ed0/external/io_grpc_grpc_java/java_grpc_library.bzl:76:14: in srcs attribute of @io_grpc_grpc_java//services:_channelz_java_grpc: Proto source with label @io_grpc_grpc_proto//:channelz_proto should be in same package as consuming rule
```

Reply:
No, those will not be fixed and is off-topic for this thread. Those are quite old.